expressions/methods.md

```diff
@@ -228,6 +228,32 @@ primitive Foo
     f.>add_option(o1).>add_option(o2).make_object() // Works. The expression returns an Object
 ```
 
+## Anonymous methods
+
+Pony has Anonymous methods (or Lambdas). They look like this :
+
+```pony
+/* Print
+1
+3
+*/
+
+use "collections"
+
+actor Main
+  new create(env:Env) =>
+    let listOfNumbers = List[U32].from([1; 2; 3; 4])
+    let filteringLambda = {(n: U32): Bool => n % 2 == 1}
+    let onlyOdd = listOfNumbers.filter(filteringLambda).values()
+    while onlyOdd.has_next() do
+      try
+        env.out.print(onlyOdd.next()?.string())
+      end
+    end
+```
+
+They are presented more in-depth in the Object Literals section.
```

Comment:
Pls make `Object Literals section` a link to the actual section: `(Object Literals section)[../expressions/object-literals.md]`

Reply:
Shouldn't be [Object Literals section](../expressions/object-literals.md) instead (swapping parenthesis and square brackets)